pkg/cmd/render/render_test.go

```diff
@@ -97,12 +98,28 @@ spec:
   serviceNetwork:
     - 2001:db8::/32
 status: {}
+`
+	infraConfig = `
+apiVersion: config.openshift.io/v1
+kind: Infrastructure
+metadata:
+  name: cluster
+spec:
+  cloudConfig:
+    name: ""
+status:
+  platform: AWS
```

Comment:
can we have a test case where platform is empty?

Reply:
added it in the Platform Unit test. PTAL